homeworks/nikolay.krasniuk_Kingstar22/9-animations/script.js

```diff
@@ -0,0 +1,63 @@
+const themeBtnEL = document.querySelector('[data-btn]');
+const hamburgerEl = document.querySelector('[data-header-burger]');
+const menuEl = document.querySelector('[data-menu]');
+const fixedMenuEl = document.querySelector('body');
+const scrollTopEl = document.querySelector('[data-scroll-btn]');
+const animationImagesEl = document.querySelectorAll('.img-animation');
+const loaderEl = document.querySelector('[data-loader]');
+
+themeBtnEL.addEventListener('click', () => {
+    const icon = document.querySelector('[data-icon]');
+    const value = icon.getAttribute('xlink:href');
+    if (value !== '#moon') {
```

Comment:
In this way, you're bound to HTML values. In some cases, you can have the wrong icon for the black theme, which will work poorly.

I suggest you create a variable that will describe the current theme and, depending on its value, set the icon and theme.